examples/engines/ray_example.py

```diff
@@ -26,7 +26,7 @@ def transform(self, df: pd.DataFrame) -> pd.DataFrame:
 def execute(token: str, logger: Callable, worker_number: int = 0, *args, **kwargs):
     config = decode_workflow_token(args.workflow_token)
 
-    workflow_id = config.get("workflow_id", str(uuid.uuid4()))
```

Comment:
This is still `job_id`

Reply:
ty

Reply:
Each running instance of a `workflow` is a `job`